doc/index.rst

```diff
@@ -721,6 +721,37 @@ Functions
 
    .. versionadded:: 4.1.0
 
+
+Helper classes
+~~~~~~~~~~~~~~
+
+.. class:: Doc(documentation, /)
```

Comment:
I'm not sure this is the right syntax in Sphinx to show that it's a positional-only parameter, I assumed so inferring from `dataclass_transform(*, eq_default=False, etc=True)`.

Reply:
Tip: enable docs previews for PRs:

https://docs.readthedocs.io/en/stable/pull-requests.html